.github/pull_request_template.md

```diff
@@ -1,18 +1,20 @@
+# Description
+
 <!--
 What is the purpose of this PR?
 
-* What is the context surrounding this PR? Include links if possible.
-* What kind of feedback do you want?
 * Have you [formatted the PR title](https://github.com/pinterest/gestalt/#releasing)? `ComponentName: Description`
 -->
 
-## Test Plan
+## Links
```

Comment:
nit: can we make these `###`? `##` seems jarringly large

Reply:
against my better a11y judgement, yes :P